# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/all.js
# Commit: 83e5baac01a9
# Full Hash: 83e5baac01a9ca24f16d971a7a89aa32650bdf3c
# Author: Tom Ritter <tom@mozilla.com>
# Date: 2021-08-09 21:33:53
# Regressor Bug: 1723204
# File Overlap Count: 1
# Description:
#   Bug 1723204: Wire up a crash for Javascript Load Telemetry r=ckerschb,Gijs
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D121417
# ==============================================================================

diff -r 2ee6c94f35b3 -r 83e5baac01a9 modules/libpref/init/all.js
--- a/modules/libpref/init/all.js	Mon Aug 09 14:42:31 2021 +0000
+++ b/modules/libpref/init/all.js	Mon Aug 09 14:42:31 2021 +0000
@@ -213,6 +213,9 @@
 pref("security.osreauthenticator.password_last_changed_lo", 0);
 pref("security.osreauthenticator.password_last_changed_hi", 0);
 
+pref("security.crash_tracking.js_load_1.prevCrashes", 0);
+pref("security.crash_tracking.js_load_1.maxCrashes", 1);
+
 pref("general.useragent.compatMode.firefox", false);
 
 pref("general.config.obscure_value", 13); // for MCD .cfg files
